airflow/www/templates/airflow/dag.html

```diff
@@ -54,40 +54,24 @@ <h4 class="pull-right">
               Back to {{ dag.parent_dag.dag_id }}</a>
           </li>
       {% endif %}
+      <li><a href="{{ url_for("airflow.trigger_dag", dag_id=dag.dag_id, root=root) }}">
+          <span class="glyphicon glyphicon-play-circle" aria-hidden="true"></span>
+        Trigger Dag
+      </a></li>
       <li><a href="{{ url_for("airflow.graph", dag_id=dag.dag_id, root=root) }}">
           <span class="glyphicon glyphicon-certificate" aria-hidden="true"></span>
         Graph View</a></li>
       <li><a href="{{ url_for("airflow.tree", dag_id=dag.dag_id, num_runs=25, root=root) }}">
           <span class="glyphicon glyphicon-tree-deciduous" aria-hidden="true"></span>
         Tree View
       </a></li>
-      <li><a href="{{ url_for("airflow.duration", dag_id=dag.dag_id, days=30, root=root) }}">
-          <span class="glyphicon glyphicon-stats" aria-hidden="true"></span>
-          Task Duration
-      </a></li>
-      <li><a href="{{ url_for("airflow.tries", dag_id=dag.dag_id, days=30, root=root) }}">
-          <span class="glyphicon glyphicon-duplicate" aria-hidden="true"></span>
-          Task Tries
-      </a></li>
-      <li>
-        <a href="{{ url_for("airflow.landing_times", dag_id=dag.dag_id, days=30, root=root) }}">
-          <span class="glyphicon glyphicon-plane" aria-hidden="true"></span>
-          Landing Times
-        </a>
-      </li>
       <li>
         <a href="{{ url_for("airflow.gantt", dag_id=dag.dag_id, root=root) }}">
           <span class="glyphicon glyphicon-align-left" aria-hidden="true"></span>
           <i class="icon-align-left"></i>
           Gantt
```

Comment:
Gantt may be removed.

Reply:
don't you think the gantt can be useful? I think it's an easy way of getting an idea of time per task or even time per pipeline.

Reply:
I think the gantt is a nice overview of task time so I've left it in, but we can always change it later.